EntheosAI

```diff
@@ -5,6 +5,9 @@
         "Entheos"
     ],
     "policies": [
+        '8b02682be393468d407580dd300ce40c4ee9d5720134941f65dd6d58',
```

Comment:
please fix up the quotation marks at line 8 :)

Reply:
Oh just saw this.. Thank you !